samples/hasura/compose.dev.yaml

```
@@ -1,6 +1,6 @@
 services:
   postgres:
-    image: postgres:16
+    image: postgres:14
```

No reason to downgrade this right now. the latest version is 17.
```suggestion
    image: postgres:16
```